app/exporters/protocol_subscription_exporter.rb

```diff
@@ -37,7 +37,7 @@ def add_more_fields(vals, protocol_subscription)
       vals['end_date'] = format_datetime(protocol_subscription.end_date)
       vals['protocol'] = protocol_subscription.protocol.name
       vals['informed_consent_given_at'] = format_datetime(protocol_subscription.informed_consent_given_at)
-      vals['filling_out_for_person_id'] = calculate_hash(protocol_subscription.filling_out_for_id)
+      vals['filling_out_for_person_id'] = protocol_subscription.filling_out_for.external_identifier
```

Comment:
Deze calculate hash functie staat nog gedefinieerd maar niet gebruikt (en ook niet getest trouwens) in app/exporters/exporters.rb. Als je die nog even weghaalt schoont dat de code ook weer wat op.